staging/src/k8s.io/component-base/metrics/legacyregistry/registry.go

```diff
@@ -53,9 +53,6 @@ func init() {
 // Handler returns an HTTP handler for the DefaultGatherer. It is
 // already instrumented with InstrumentHandler (using "prometheus" as handler
 // name).
-//
-// Deprecated: Please note the issues described in the doc comment of
```

Comment:
I am not sure to understand why the changes from https://github.com/kubernetes/kubernetes/commit/9ead9373f350c7ad438257a9e8b8977a67b900eb remove the deprecation from this function, could you please explain?

The issues mentioned in https://github.com/prometheus/client_golang/blob/v1.11.0/prometheus/promhttp/http.go#L216-L232 are still very valid to me.

Reply:
Yes after your explanation, it indeed makes a lot of sense to remove this comment. Thanks!